dockerfiles/alpine/Dockerfile

```
@@ -39,23 +49,23 @@ RUN echo "fn main() {println!(\"if you see this, the build broke\")}" > bob/src/
   && echo "fn main() {println!(\"if you see this, the build broke\")}" > bob-apps/bin/brt.rs \
   && echo "fn main() {println!(\"if you see this, the build broke\")}" > bob-apps/bin/bobt.rs \
   && echo "fn main() {println!(\"if you see this, the build broke\")}" > bob-access/src/lib.rs \
-  && cargo build --release --target=x86_64-unknown-linux-musl
+  && cargo build --profile=${BUILD_PROFILE:-"release"} --target=x86_64-unknown-linux-musl
 
 
 # separate stage for proto build
 RUN echo "fn main() {println!(\"if you see this, the build broke\")} pub mod grpc {include!(\"bob_storage.rs\");}" > bob-grpc/src/lib.rs \
   && mkdir -p bob-grpc/proto
 COPY bob-grpc/proto/* bob-grpc/proto/
 COPY bob-grpc/build.rs bob-grpc/build.rs
-RUN cargo build --release --target=x86_64-unknown-linux-musl \
-  && rm -f target/x86_64-unknown-linux-musl/release/deps/bob* \
-  && rm -f target/x86_64-unknown-linux-musl/release/deps/libbob* 
+RUN cargo build --profile=${BUILD_PROFILE:-"release"} --target=x86_64-unknown-linux-musl \
+  && rm -f target/x86_64-unknown-linux-musl/${BUILD_PROFILE_DIR:-"release"}/deps/bob* \
+  && rm -f target/x86_64-unknown-linux-musl/${BUILD_PROFILE_DIR:-"release"}/deps/libbob* 
```

For `BUILD_PROFILE_DIR` default value is also not needed, because you estimate it earlier
```suggestion
  && rm -f target/x86_64-unknown-linux-musl/${BUILD_PROFILE_DIR}/deps/libbob* 
```